docs/explanation/core/running-in-production.md

```diff
@@ -8,8 +8,8 @@ order: 4
 
 By default, CometBFT uses the `syndtr/goleveldb` package for its in-process
 key-value database. If you want maximal performance, it may be best to install
-the real C-implementation of LevelDB and compile CometBFT to use that using
-`make build COMETBFT_BUILD_OPTIONS=cleveldb`. See the [install
+RocksDB and compile CometBFT to use that using
+`make build COMETBFT_BUILD_OPTIONS=rocksdb`. See the [install
```

Comment:
Is this suggestion based on any evidence that RocksDB provides the maximal performance ? I understand in the past we suggested `cleveldb` because of the `C-Implementation` (not sure if that was also an accurate statement), but maybe we should re-word the `maximal performance` to something like `one of the best performance` or something like this. I don't think we have any benchmarks to support this statement (afaik we didn't test one against the other and we even included Pebble recently that might provide good performance too).

Reply:
@melekes , @andynog , just seeing this. I wonder whether Andy is right and we might want to suggest people to use either rocksdb or pebble. Or we could run the benchmarks on rocksdb to be sure of this. We could also link to the storage report and report the fact that pebble does good compaction vs goleveldb.

Reply:
@jmalicevic what you're suggesting sounds good to me. The changes above to the documentation were more like a quick patch.

Reply:
I just saw that this text was actually removed with a subsequent PR. I can open a small issue to extend this with the one line saying there are multiple options and to look for these two as well for better performance. But its low priority now that we are not explicitly suggesting rocksdb.